pow/src/main/java/tech/pegasys/teku/pow/Eth1DepositManager.java

```diff
@@ -146,8 +146,16 @@ private void preGenesisSubscription(final BigInteger lastProcessedBlockNumber) {
     return eth1Provider
         .getLatestEth1Block()
         .thenApply(EthBlock.Block::getNumber)
-        .thenApply(number -> number.subtract(Constants.ETH1_FOLLOW_DISTANCE.bigIntegerValue()))
         .thenApply(UInt64::valueOf)
+        .thenApply(
+            number -> {
+              if (number.isLessThan(Constants.ETH1_FOLLOW_DISTANCE)) {
```

Comment:
Should we retry less frequently and log a warning in this case?

Reply:
Makes sense, will do that.

Reply:
Not sure we even need a warning - it's a pretty expected condition when using a local test chain.  And we could just start following from 0 now I think about it.  The genesis block of ETH1 is finalized so there's no risk of it being re-orged.  We should then just sit at block 0 until the chain reaches 1024 blocks long where we can carry on as normal.

Reply:
> We should then just sit at block 0 until the chain reaches 1024 blocks long where we can carry on as normal.

That made sense to me, so I made the changes to reflect that.